plugins/tokens/update.js

```diff
@@ -58,11 +58,11 @@ module.exports = () => ({
                             });
 
                             return token.update()
-                            .then(() => {
-                                const output = token.toJson();
+                                .then(() => {
```

Comment:
Ah, I must have forgotten to refactor that when I got rid of the `delete token.hash`